ext/standard/password.c

```diff
@@ -538,7 +538,7 @@ PHP_FUNCTION(password_hash)
 					ZSTR_VAL(out),
 					ZSTR_LEN(out),
 					ZSTR_VAL(encoded),
-					ZSTR_LEN(encoded),
+					ZSTR_LEN(encoded) + 1,
```

Comment:
Perhaps could just put encoded_len here? But anyway seems you've nailed the case.

Thanks.

Reply:
Ah, yes, `encoded_len` appears to be better. Thanks. I'm going to *apply* the modified PR against PHP-7.2.